src/mmw/sass/components/_modals.scss

```diff
@@ -31,7 +31,7 @@
 .modal-basic {
 
   .modal-content {
-    padding: 1rem;
+    padding: 3rem;
```

Comment:
This affects the other modals, but for the better (?) @jfrankl, this cool? 

Before (`1rem`)

![screen shot 2017-06-14 at 4 50 05 pm](https://user-images.githubusercontent.com/7633670/27154048-a756421e-5121-11e7-8e8c-c2df8d7c0e72.png)
![screen shot 2017-06-14 at 4 50 14 pm](https://user-images.githubusercontent.com/7633670/27154049-a759d2e4-5121-11e7-9080-02931987a1de.png)

After (`3rem`)
![screen shot 2017-06-14 at 4 47 43 pm](https://user-images.githubusercontent.com/7633670/27153976-5fd4c38e-5121-11e7-8d77-b641111ad79d.png)
![screen shot 2017-06-14 at 4 50 28 pm](https://user-images.githubusercontent.com/7633670/27154038-9dcb3894-5121-11e7-86b8-816fbe013054.png)